chain/client/src/stateless_validation/state_witness_actions.rs

```diff
@@ -224,11 +224,54 @@ impl StateWitnessActions {
         Ok(())
     }
 
+    /// Function to validate the partial encoded state witness. We check the following
+    /// - shard_id is valid
+    /// - we are one of the validators for the chunk
+    /// - part_ord is valid and within range of the number of expected parts for this chunk
+    /// - partial_witness signature is valid and from the expected chunk_producer
+    /// TODO(stateless_validation): Include checks from handle_orphan_state_witness in orphan_witness_handling.rs
+    /// These include checks based on epoch_id validity, witness size, height_created, distance from chain head, etc.
     fn validate_partial_encoded_state_witness(
```

Comment:
This function is heavily inspired from `partially_validate_state_witness` in chain/client/src/stateless_validation/chunk_validator/mod.rs

We would eventually get rid of `partially_validate_state_witness`